module/purpose/Soccer/src/Soccer.cpp

```diff
@@ -87,7 +87,7 @@ namespace module::purpose {
                 log<NUClear::INFO>("Force playing started.");
                 cfg.force_playing = true;
                 emit<Task>(std::make_unique<FindPurpose>());
-                emit<Task>(std::unique_ptr<StandStill>(nullptr));
+                // emit<Task>(std::unique_ptr<StandStill>(nullptr));
```

Comment:
Commented out?? Future development?

Reply:
see second dot point in the main PR comment